src/context.js

```diff
@@ -1,3 +1,21 @@
-import React from 'react';
+import React, { useContext } from 'react';
 
-export const { Consumer, Provider } = React.createContext([]);
+const FeaturesContext = React.createContext([]);
```

Comment:
I exported the Consumer/Provider as `FeaturesConsumer`/`FeaturesProvider`, but I suppose we should export the Context instead (since `useContext` wants it, presumably other API's might too), and its easy enough to FeatureTogglesContext.Consumer as a client.